examples/decentralized_protocol_config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"environment": {
+		"type": "OrdinaryBandit",
+		"OrdinaryBandit": {
+			"arm": "BernoulliArm",
+			"means": [0.7, 0.5, 0.3, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
+		},
+		"protocol": "DecentralizedProtocol",
+		"DecentralizedProtocol": {
+			"num_players": 10
+		}
+	},
+	"learner": {
+		"goal": "regretmin",
+		"type": "decentralizedlearner",
```

Comment:
How about using the name `decentralizedordinarylearner` to indicate the bandit model?